__e2e__/root.test.ts

```diff
@@ -0,0 +1,62 @@
+import path from 'path';
+
+import {
+  spawnScript,
+  runCli,
+  getTempDirectory,
+  cleanup,
+  writeFiles,
+} from '../jest/helpers';
+
+const cwd = getTempDirectory('test_different_roots');
+
+beforeAll(() => {
+  // Register all packages to be linked
+  for (const pkg of ['platform-ios', 'platform-android']) {
+    spawnScript('yarn', ['link'], {
+      cwd: path.join(__dirname, `../packages/${pkg}`),
+    });
+  }
+
+  // Clean up folder and re-create a new project
+  cleanup(cwd);
+  writeFiles(cwd, {});
+
+  // Initialise React Native project
+  runCli(cwd, ['init', 'TestProject']);
+
+  // Link CLI to the project
+  for (const pkg of [
+    '@react-native-community/cli-platform-ios',
+    '@react-native-community/cli-platform-android',
+  ]) {
+    spawnScript('yarn', ['link', pkg], {
```

Comment:
you can run `yarn link` with multiple packages, no need for a for-loop here

Reply:
nice, didn't notice `...` in docs!